src/org.hdfgroup.hdfview/hdf/view/TableView/DefaultCompoundAttributeTableView.java

```diff
@@ -0,0 +1,33 @@
+/*****************************************************************************
+ * Copyright by The HDF Group.                                               *
+ * Copyright by the Board of Trustees of the University of Illinois.         *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of the HDF Java Products distribution.                  *
+ * The full copyright notice, including terms governing use, modification,   *
+ * and redistribution, is contained in the files COPYING and Copyright.html. *
+ * COPYING can be found at the root of the source code distribution tree.    *
+ * Or, see https://support.hdfgroup.org/products/licenses.html               *
+ * If you do not have access to either file, you may request a copy from     *
+ * help@hdfgroup.org.                                                        *
+ ****************************************************************************/
+
+package hdf.view.TableView;
+
+import java.util.HashMap;
+
+import hdf.view.DataView.DataViewManager;
+
+public class DefaultCompoundAttributeTableView extends DefaultCompoundDSTableView implements TableView {
+
+    private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(DefaultCompoundAttributeTableView.class);
+
+    public DefaultCompoundAttributeTableView(DataViewManager theView) {
+        this(theView, null);
+    }
+
+    @SuppressWarnings("rawtypes")
+    public DefaultCompoundAttributeTableView(DataViewManager theView, HashMap dataPropertiesMap) {
+        super(theView, dataPropertiesMap);
```

Comment:
Considering that there's nothing in this file, it probably isn't even needed. Since a compound attribute should be an extension of CompoundDS, the compound DS tableview should be able to be reused.

Reply:
Seems to make sense - not sure why I had created this.

Reply:
The Scalar version also has the same issues.
Both can be deleted.